packages/react/src/AnchoredOverlay/AnchoredOverlay.test.tsx

```diff
@@ -50,6 +54,12 @@ const AnchoredOverlayTestComponent = ({
 }
 
 describe('AnchoredOverlay', () => {
+  implementsClassNameBehavior(
+    AnchoredOverlay,
+    'prc-Overlay-Overlay-ViJgm',
+    component => component.container.firstChild!.childNodes[1].firstChild?.firstChild as HTMLElement,
+    props => <AnchoredOverlayTestComponent initiallyOpen={true} {...props} />,
+  )
```

Comment:
my dream test function call looks like this:

```tsx
implementsClassName(ActionList)

// or if we need to,

implementsClassName(props => <AnchoredOverlayFixture {...props} />)
```

`implementsClassName` should
1. render without custom className to figure out the base class and base element
2. render with custom className, make sure it has both classes on the base element

Reply:
Update: yes we can look for the className, cleaned it up to where we're just passing the render fixture and the className, I'm not sure about trying to infer the className just by looking for prc-* because of what I said ^ regarding composed components, let me know what you think!

Reply:
Ohhh, righto! I don't know why I understand this earlier.

> so even knowing the full className, I don't know how to find the element is my main problem

Is this still valid?